staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/validation.go

```diff
@@ -236,16 +262,23 @@ func validateExtensions(x *Extensions, fldPath *field.Path) field.ErrorList {
 }
 
 // validateValueValidation checks the value validation in a structural schema.
-func validateValueValidation(v *ValueValidation, skipAnyOf, skipFirstAllOfAnyOf bool, lvl level, fldPath *field.Path) field.ErrorList {
+func validateValueValidation(v *ValueValidation, skipAnyOf, skipFirstAllOfAnyOf bool, lvl level, fldPath *field.Path, opts ValidationOptions) field.ErrorList {
 	if v == nil {
 		return nil
 	}
 
 	allErrs := field.ErrorList{}
 
+	// We still unconditionally forbid XValidations in quantifiers, the only
+	// quantifier that is allowed to have right now is AllOf. This is due to
+	// implementation constraints - the SchemaValidator would need to become
+	// aware of CEL to properly implement the other quantifiers.
+	optsWithCELDisabled := opts
+	optsWithCELDisabled.AllowNestedXValidations = false
```

Comment:
Im not sure encoding XValidations as a `PositiveNestedValueValidation` is best, since the reason we arent including it in other quantifiers with negations is simply an implementation detail rather than anything about CEL expressions as validations themselves.